### PR TITLE
Add DOHCOIN (DOH) on Base

### PR DESCRIPTION
**Pull request by budezllc**
Adding DOHCOIN on Base List

Basescan: https://basescan.org/token/0xf37923311d71c986d701cddb3394777322294e3a

@roberto-bayardo

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
Is this a native L2 token?  If so can you add `"nobridge":  true` to the data.json to fix the validation error?

File: data/DOH/data.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "DOHCOIN",
+    "symbol": "DOH",
+    "decimals": 18,
+    "description": "DOHCOIN Token mined with DOH MINER NFTs.",
+    "website": "https://dohcoin.com",
+    "twitter": "@dohwtf",
+    "tokens": {
+      "base": {
+        "address": "0xF37923311D71c986D701cddb3394777322294E3a"
```
Comment (roninjin10) [inline data/DOH/data.json:10]: I notice there is no ethereum address. Can you clarify if this is a native l2 token that doesn't have an l1 address? If so we need to fix our validation to handle this

Comment (budezllc) [reply data/DOH/data.json:10]: This is a native l2 base token only.

Comment (hamdiallam) [reply data/DOH/data.json:11]: thanks for clarifying!

**Pull request closed without merging**